webrtc.html

```diff
@@ -720,7 +720,7 @@
                   </li>
 
                   <li>
-                    <p>Fire a simple event named <code><a href=
+                    <p>Queue a task to fire a simple event named <code><a href=
```

Comment:
We want the state update to happen synchronously as the event that tells the script about it, so we don't want to queue an other task here. @jan-ivar, does it matter if we fire the event first or reject the promise? I.e. change the order of step 6 and 7 in this algorithm.